packages/app-store/office365video/lib/VideoApiAdapter.ts

```diff
@@ -153,6 +153,10 @@ const TeamsVideoApiAdapter = (credential: CredentialPayload): VideoApiAdapter =>
 
       const resultObject = JSON.parse(resultString);
 
+      if (!resultObject.id || !resultObject.joinUrl) {
+        throw new HttpError({ statusCode: 500, message: "Error creating MS Teams meeting" });
+      }
+
```

Comment:
if resultObject returned 204 status it's body will be {}, so doing resultObject.id will return as undefined and not returning error to main function createMeeting()